app/uk/gov/hmrc/agentsubscriptionfrontend/models/ChainedSessionDetails.scala

```diff
@@ -19,7 +19,10 @@ package uk.gov.hmrc.agentsubscriptionfrontend.models
 import play.api.libs.json.Json
 import uk.gov.hmrc.agentmtdidentifiers.model.Utr
 
-case class ChainedSessionDetails(knownFacts: KnownFactsResult, wasEligibleForMapping: Option[Boolean])
+case class ChainedSessionDetails(
+  knownFacts: KnownFactsResult,
```

Comment:
Need a future tech debt ticket to remove this `KnownFactsResult` from this `ChainedSessionDetails`. It's not all requried beyond creating clean creds in the journey.